neo/rawio/axonarawio.py

```diff
@@ -333,6 +339,100 @@ def _get_analogsignal_chunk(self, block_index, seg_index, i_start, i_stop,
 
         return raw_signals
 
+    def _get_mock_analogsignal_chunk(self, channel_ids, start_frame, end_frame,
```

Comment:
@sbuergers I am not sure if I would put such a preprocessing function into the axona module as in principle the need for padding / concatenating waveforms might occur independent of the format. E.g. for analogisignals there is a concatenation method  implemented here https://github.com/NeuralEnsemble/python-neo/blob/b4def58e3ecb34c7a4e57b68b64302cc8ab43b68/neo/core/analogsignal.py#L661 . Would the addition of a corresponding waveform padding method, e.g. as spiketrain method be a good solution for you too?

Reply:
@JuliaSprenger Thanks for the feedback, I was wondering what solution would be appropriate / sensible, since some form of analogsignal should be returned. 

I do not have a strong preference as to what the solution should be, the `AxonaUnitRecordingExtractor` does not depend on the analogsignal at all. Shall we leave this solution until a better one is found? I really want to wrap up the Hussaini project. ;-)

Reply:
Hi @sbuergers! I see you don't want to depend on the neo core for this. Still since this method is not memory mapping itself it does not necessariliy be part of the AxonaRawIO, but I am not sure yet on how to handle this in a better way without moving it to core.

Another note: I think due to the randomly generated samples the code will provide different traces for overlapping frames (or if you request the same frame twice). Will this be an issue for you later? If this is the case then I think this would be anyway worth mentioning in the doc to avoid confusion in the future.

Reply:
Hi again! I just talked to @samuelgarcia and he also thinks that this method should not be part of a specific Neo RawIO. So would it be possible to either make it more generic and include it on the SpikeInterface side of your project or as a utility function within your project?

Reply:
Hi @JuliaSprenger, I may have misunderstood your initial comment. I do not mind moving the padding / filling with noise to `core`. To avoid confusion about the noise it might also be good to set the default standard deviation to 0, which is replicable and more intuitive for a user who knows the `.X` format.